android/src/com/mapswithme/maps/routing/SlotFrame.java

```diff
@@ -99,7 +100,7 @@ private void updateText()
       if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
         mText.setText(R.string.p2p_your_location);
       else
-        mText.setText(mMapObject.getTitle());
+        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
```

Comment:
Всё верно, и в таком случае условие будет false и в слоте будет показан тайтл. Сабтайтл показывается ТОЛЬКО для тех объектов, где тайтл пустой.
> On Mar 25, 2016, at 6:54 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/routing/SlotFrame.java <https://github.com/mapsme/omim/pull/2535#discussion_r57457095>:
> 
> > @@ -99,7 +100,7 @@ private void updateText()
> >        if (MapObject.isOfType(MapObject.MY_POSITION, mMapObject))
> >          mText.setText(R.string.p2p_your_location);
> >        else
> > -        mText.setText(mMapObject.getTitle());
> > +        mText.setText(!TextUtils.isEmpty(mMapObject.getTitle()) ? mMapObject.getTitle() : mMapObject.getSubtitle());
> Dropped Pin будет приходить из ядра в Title. Subtitle будет пустой.
> … <x-msg://35/#>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2535/files/de732bc2b8309d0c9fd4efeb458f1d298b1231f0#r57457095>